### PR TITLE
31/convert to package

### PR DESCRIPTION
**Pull request by Deefoozy**
Changed project structure and name of the project.

This is being done as i want to make this project a library.

### CODE REVIEW

File: WebSocketiny/WebSocketiny.csproj [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+		<LangVersion>8</LangVersion>
```
Comment (LDSpits) [inline WebSocketiny/WebSocketiny.csproj:4]: You can optionally enable nullability alalyzers if you want
```xml
<Nullable>enable</Nullable>
```

**Review by LDSpits — DISMISSED** [review]
Delete is gud.
Seems to be a bit much though, did you start over or something?

File: WebSocketiny/WebSocketiny.csproj (hunk 1 above)
Comment (Deefoozy) [reply WebSocketiny/WebSocketiny.csproj:4]: I'll blame you if borked!

**Review by LDSpits — APPROVED** [review]
LGTM

**Pull request merged**